Java/pom.xml

```diff
@@ -7,8 +7,8 @@
     <artifactId>aircompany</artifactId>
     <version>1.0-SNAPSHOT</version>
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>3.6.1</maven.compiler.source>
```

Comment:
why you set 3.6.1? 
https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html

Reply:
I was not able to run project without this change.
At first, a got error error:java: error: release version 5 not supported -> changed in project settings Java Compiler from 1.5 to 11
Then I got an error:java: source option 5 is no longer supported. use 6 or later.
Read in https://github.com/jflex-de/jflex/issues/400 that this error can be fixed by lowering JAVA version to 8 or using not so old maven compiler. So, in general, I could use 3.8.1 version also. 
I just haven't checked what maven version was the latest